pytest_reportportal/listener.py

```diff
@@ -1,39 +1,66 @@
+"""RPReportListener define built-in pytest functions."""
 import pytest
 import logging
+
 try:
-    from html import escape   # python3
+    from html import escape  # python3
 except ImportError:
-    from cgi import escape    # python2
-
+    from cgi import escape  # python2
 
 try:
     # This try/except can go away once we support pytest >= 3.3
     import _pytest.logging
+
     PYTEST_HAS_LOGGING_PLUGIN = True
     from .rp_logging import RPLogHandler, patching_logger_class
 except ImportError:
     PYTEST_HAS_LOGGING_PLUGIN = False
 
 
 class RPReportListener(object):
+    """RPReportListener class."""
+
     def __init__(self, py_test_service,
                  log_level=logging.NOTSET,
                  endpoint=None):
+        """
+        Initialize RPReport Listener instance.
+
+        :param py_test_service: PyTestServiceClass instance
+        :param log_level ('CRITICAL','ERROR', 'WARNING','INFO',
+        'DEBUG','NOTSET')
+        :param endpoint: http link
+        """
         # Test Item result
         self.PyTestService = py_test_service
         self.result = None
         self.issue = {}
         self._log_level = log_level
         if PYTEST_HAS_LOGGING_PLUGIN:
-            self._log_handler = RPLogHandler(py_test_service=py_test_service,
-                                             level=log_level,
-                                             filter_reportportal_client_logs=True,
-                                             endpoint=endpoint)
+            self._log_handler = \
+                RPLogHandler(py_test_service=py_test_service,
+                             level=log_level,
+                             filter_reportportal_client_logs=True,
+                             endpoint=endpoint)
 
     @pytest.hookimpl(hookwrapper=True)
     def pytest_runtest_protocol(self, item):
-        # Adding issues id marks to the test item
-        # if client doesn't support item updates
+        """
+        Adding issues id marks to the test item.
+
+        if client doesn't support item updates
+        :param item:    {name:'',
```

Comment:
It is an incorrect item description. item is the instance of the Pytest Item class.